apps/site/test/site/lib/react_test.exs

```diff
@@ -64,38 +56,11 @@ defmodule Site.ReactTest do
       log =
         CaptureLog.capture_log(fn ->
           assert "" ==
-                   React.render("TransitNearMe", %{
-                     query: %{},
-                     mapId: "map-id",
-                     mapData: %{markers: []},
-                     routeSidebarData: "crash",
-                     stopSidebarData: []
-                   })
+                   React.render("TransitNearMe", %{})
         end)
 
-      assert log =~ ~r/react_renderer component=TransitNearMe.* is not a function/
-    end
-
-    test "it translates zero-width space HTML entities into UTF-8 characters", %{
```

Comment:
Did you mean to remove this test?

Reply:
It needs to be moved somewhere else because there is no data w/ a zero-width space that would be rendered on the page initially.

I'm thinking to add a new task to the backlog to put this test into another component. I don't think it is specific to TNM, but that is where we first encountered this issue.